lib/unwrappr/cli.rb

```diff
@@ -62,38 +67,58 @@ def execute
       option(['-r', '--repo'],
              'REPO',
              <<~DESCRIPTION,
-               a repo in github <owner/project>, may be specified multiple times
+               a repo in GitHub <owner/project>, may be specified multiple times
              DESCRIPTION
              required: true,
              multivalued: true)
 
+      option ['-R', '--recursive'],
+             :flag,
+             'Recurse into subdirectories',
+             attribute_name: :recursive
+
       def execute
         repo_list.each do |repo|
-          unless Dir.exist?(repo)
-            GitCommandRunner.clone_repository(
-              "https://github.com/#{repo}",
-              repo
-            )
-          end
+          GitCommandRunner.clone_repository("https://github.com/#{repo}", repo) unless Dir.exist?(repo)
 
-          Dir.chdir(repo) { Unwrappr.run_unwapper_in_pwd(base_branch: base_branch, lock_files: lock_files) }
+          Dir.chdir(repo) do
+            Unwrappr.run_unwrappr_in_pwd(base_branch: base_branch, lock_files: lock_files, recursive: recursive?)
+          end
         end
       end
     end
   end
 
-  def self.run_unwapper_in_pwd(base_branch:, lock_files:)
+  def self.run_unwrappr_in_pwd(base_branch:, lock_files:, recursive:)
     return unless any_lockfile_present?(lock_files)
```

Comment:
I notice this check for lock-files does this mean the recursion won't run unless there's a lock file in the root of the git repository?

Reply:
Good spot. It does. I think that's OK.